### PR TITLE
Add support for config.sslkeylogfile to log TLS keys

### PR DESCRIPTION
**Pull request by manuelm**

### CODE REVIEW

File: cyclone/cyclone_ssl/tls_key_material.c
Comment (SciLor) [inline cyclone/cyclone_ssl/tls_key_material.c:1]: The cyclone framework shouldn't be changed.
Please copy the original file to src/cyclone/cyclone_ssl/tls_key_material.c/h and commit it. Then save it to enforce the auto format and commit again. Then you can integrate your change and add it to the makefile.

Comment (manuelm) [reply cyclone/cyclone_ssl/tls_key_material.c:1]: Another aproach would be to disable `stringop-truncation`-warnings for the cyclon framework.

Comment (SciLor) [reply cyclone/cyclone_ssl/tls_key_material.c:1]: Is this just a warning or does it prevent you from building it entirely?

Comment (manuelm) [reply cyclone/cyclone_ssl/tls_key_material.c:1]: Due to `Werror` its a compilation error. Otherwise it's a warning.

Comment (SciLor) [reply cyclone/cyclone_ssl/tls_key_material.c:1]: @g3gg0 what to you think about it?

**Comment by manuelm** [comment]
as discussed implemented as `core.sslkeylogfile` now

**Pull request merged**